src/modules/loadLeaderboard.js

```diff
@@ -0,0 +1,14 @@
+export default function loadLeaderboard(array) {
+  function loop(data) {
+    return `
+          <tr>
+            <td>${data.user}</td>
+            <td>${data.score}</td>
+          </tr>
+        `;
+  }
```

Comment:
As part of the project requirements, you are supposed to use the arrow functions instead of the function keyword. Kindly change this and all other function to use the arrow functions. :pray: